.github/workflows/json-schema-synapse.yml

```diff
@@ -0,0 +1,199 @@
+        
+        # FollowUp
+        poetry run python scripts/linkml_to_flat_synapse_jsonschema.py \
+          "modules/Clinical/domains/followup.yaml" \
+          --class-name "FollowUp" \
+          --output "HTAN.FollowUp-${{ steps.version.outputs.version }}-schema.json"
+        
+        # MolecularTest
+        poetry run python scripts/linkml_to_flat_synapse_jsonschema.py \
+          "modules/Clinical/domains/molecular.yaml" \
+          --class-name "MolecularTest" \
+          --output "HTAN.MolecularTest-${{ steps.version.outputs.version }}-schema.json"
+        
+        # Exposure
+        poetry run python scripts/linkml_to_flat_synapse_jsonschema.py \
+          "modules/Clinical/domains/exposure.yaml" \
+          --class-name "Exposure" \
+          --output "HTAN.Exposure-${{ steps.version.outputs.version }}-schema.json"
+        
+        # FamilyHistory
+        poetry run python scripts/linkml_to_flat_synapse_jsonschema.py \
+          "modules/Clinical/domains/family_history.yaml" \
+          --class-name "FamilyHistory" \
+          --output "HTAN.FamilyHistory-${{ steps.version.outputs.version }}-schema.json"
+        
+        # VitalStatus
+        poetry run python scripts/linkml_to_flat_synapse_jsonschema.py \
+          "modules/Clinical/domains/vital_status.yaml" \
+          --class-name "VitalStatus" \
+          --output "HTAN.VitalStatus-${{ steps.version.outputs.version }}-schema.json"
+    
+    - name: Register schemas in Synapse
+      env:
+        SYNAPSE_USERNAME: ${{ secrets.SYNAPSE_USERNAME }}
+        SYNAPSE_AUTH_TOKEN: ${{ secrets.SYNAPSE_AUTH_TOKEN }}
+      run: |
+        # Register each generated schema
+        for schema_file in JSON_Schemas/*.json; do
+          if [ -f "$schema_file" ]; then
+            schema_name=$(basename "$schema_file" .json)
+            echo "Processing schema: $schema_name"
+            
+            poetry run python scripts/synapse_json_schema_bind.py \
+              -p "$schema_file" \
+              -n "${{ env.ORGANIZATION_NAME }}" \
+              --no_bind
+          fi
+        done
+    
+    - name: Bind schemas to Synapse folders
+      env:
+        SYNAPSE_USERNAME: ${{ secrets.SYNAPSE_USERNAME }}
+        SYNAPSE_AUTH_TOKEN: ${{ secrets.SYNAPSE_AUTH_TOKEN }}
+      run: |
+        # Bind each schema to its appropriate folder
+        # Clinical data folder (all clinical schemas go here)
+        CLINICAL_FOLDER_ID="syn69060693"  # Update with actual folder ID
+        
+        for schema_file in JSON_Schemas/*.json; do
+          if [ -f "$schema_file" ]; then
+            schema_name=$(basename "$schema_file" .json)
+            echo "Binding schema: $schema_name to Clinical folder"
+            
+            poetry run python scripts/synapse_json_schema_bind.py \
+              -p "$schema_file" \
+              -t "$CLINICAL_FOLDER_ID" \
+              -n "${{ env.ORGANIZATION_NAME }}"
+          fi
+        done
```

Comment:
This will auto-bind folders right now after the JSON schemas are bound, but maybe you don't want to automatically bump versions of jsonschemas bound to folders.  Perhaps comment this out first until we figure out more?